x-pack/platform/plugins/private/upgrade_assistant/__jest__/client_integration/es_deprecations/cluster_settings_deprecation_flyout.test.ts

```diff
@@ -16,7 +16,7 @@ describe('Cluster settings deprecation flyout', () => {
   let testBed: ElasticsearchTestBed;
   let httpRequestsMockHelpers: ReturnType<typeof setupEnvironment>['httpRequestsMockHelpers'];
   let httpSetup: ReturnType<typeof setupEnvironment>['httpSetup'];
-  const clusterSettingDeprecation = esDeprecationsMockResponse.deprecations[4];
+  const clusterSettingDeprecation = esDeprecationsMockResponse.migrationsDeprecations[4];
```

Comment:
Handling change from combined deprecations to separate deprecation and health issues.
Remaining tests also refactored to handle the change in the response from `getESUpgradeStatus`